Monika After Story/game/script-topics.rpy

```diff
@@ -4940,3 +4940,297 @@ label monika_asks_family:
                 m 1f "You can tell me about your family when you're ready, [player]." 
                 m 1j "I love you very much!"                              
     return
+    
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_fun_facts_open",category=['misc'],prompt="Fun facts",pool=True))
+
+default fun_facts_start = 0
+default persistent._mas_funfactfun = True
+
+label monika_fun_facts_open:
+    #Determines the fact number and whether it's a bad fact
+    python:
+        fun_facts_bad_chance = random.randint(1,100)
```

Comment:
While `random` is usable here, we prefer to keep things within renpy scope so we don't have to add additional `imports` somewhere. Use the `randint` [provided](https://www.renpy.org/doc/html/other.html#renpy-random) by renpy. (Which it appears that you actually did do that a few lines below).